### ISSUE TITLE
Manpage lists have too much space between items

### ISSUE THREAD

**Issue by Diegovsky** [asciidoctor/asciidoctor#4578]
Here is an example file that reproduces the issue:

```adoc
= pod(1)
:doctype: manpage
:man-linkstyle: pass:[blue R < >]

== NAME
pod - example manpage thing

== SYNOPSIS
Words 

.List:
* one
* two

.Ordered:
. one
. two
```
I expected the lines to not contain any lines around them, but it seems that lines are added around every list item. Here's the output of `asciidoctor -b manpage pod.adoc -o - | man -l`:

```
POD(1)                                                                    POD(1)

NAME
       pod - example manpage thing

SYNOPSIS
       Words

       List:

       •   one

       •   two

       Ordered:

        1. one

        2. two

                                   2024-04-23                             POD(1)

```

I expected to see something like this:

```
POD(1)                                                                    POD(1)

NAME
       pod - example manpage thing

SYNOPSIS
       Words

       List:
       •   one
       •   two

       Ordered:
        1. one
        2. two

                                   2024-04-23                             POD(1)

```

I tried to see if if was possible to change this behaviour by searching the [unordered list](https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#markers) page but it only mentions styles there.

If this is inteded behaviour, would it be possible to add a way to change this?

**Comment by mojavelinux** [comment]
This is by design for compatibility with the output produced by the DocBook toolchain. I'd be willing to consider adding support for the `compact` role on the list to reduce the space. However, someone would need to submit the pull request to add that functionality. It's not something I'm going to work on.

Another option is to extend the manpage converter and override the markup that it generates.

**Comment by Diegovsky** [comment]
I think overriding the manpage converter is better as it's probably intended for other formats to have more spacing around list items, though, I have no knowledge of Ruby so I'll wait for someone to implement this.

On an unrelated note: this project is awesome and asciidoc is a very nice format, thank you for your work on the project :)

**Comment by mojavelinux** [comment]
Thanks for your kind words!

**Comment by mojavelinux** [comment]
Let me know what you'd like to do. Like I said, I'd be open to honoring the compact role, but if you'd like to override the converter, then I won't spend time on that proposal now.

**Comment by mojavelinux** [comment]
Yes, I think it's doable. If you'd like to try and need help, please ask in the project chat at https://chat.asciidoctor.org.
